### PR TITLE
add a jupyter notebook 19A to check weighted RF

### PR DESCRIPTION
**Pull request by shifwang**
The weighted RF is now implemented in the branch [here](https://github.com/shifwang/scikit-learn/tree/feature_weight)

### CODE REVIEW

**Comment by shamindras** [comment]
@shifwang - why is this build failing?

Do we need to fix this and MRG?

File: jupyter/utils/irf_jupyter_utils.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 
 def generate_rf_example(sklearn_ds=load_breast_cancer(),
```
Comment (shamindras) [inline jupyter/utils/irf_jupyter_utils.py]: @shifwang - could we remove this from `irf_jupyter_utils`?

We can just paste this function directly into the `jupyter notebook` to ensure that the notebook can compile.

This function is currently breaking the tests and will not be used in our final implementation, so best to move it to the notebook directly than putting it here.

Comment (shifwang) [inline jupyter/utils/irf_jupyter_utils.py]: This is the function you wrote in the first place. In order to be compatible with all the other jupyter notebooks, I modified the function a little bit and passed the check.

Comment (shamindras) [inline jupyter/utils/irf_jupyter_utils.py]: @shifwang - thanks for that! Best to always have checks passing

**Pull request merged**